.github/workflows/ci.yml

```diff
@@ -188,14 +188,49 @@ jobs:
         files: mockito-integration-tests/android-tests/build/reports/coverage/android-tests/debug/connected/report.xml
         fail_ci_if_error: true
 
+  #
+  # GraalVM native tests job
+  #
+  graalvm:
+    runs-on: ubuntu-latest
+    if: "! contains(toJSON(github.event.commits.*.message), '[skip ci]')"
+
+    steps:
+    - name: 1. Check out code
+      uses: actions/checkout@v5
+      with:
+        fetch-depth: '0'
+
+    - name: 2. Setup GraalVM
+      uses: graalvm/setup-graalvm@7f488cf82a3629ee755e4e97342c01d6bed318fa
```

Comment:
Can we use a version instead of a commit hash or is nothing published?

Reply:
https://github.com/mockito/mockito/pull/3714/commits/f443b576e53f6a4dcb5d6f9f0b87f22afa7472b4